core/src/main/java/com/wttech/aem/acm/core/osgi/InstanceInfo.java

```diff
@@ -27,4 +40,22 @@ public boolean isRunMode(String runMode) {
     public Set<String> getRunModes() {
         return slingSettings.getRunModes();
     }
+
+    public InstanceSettings getInstanceSettings() {
+        boolean publish = isPublish();
+        String propUrl = bundleContext.getProperty("PRODUCTINFO_VERSION");
+        return new InstanceSettings(TimeZone.getDefault().getID(), publish, isCloudVersion(propUrl));
+    }
+
+    private boolean isCloudVersion(String version) {
+        // For local instance treat it same as on-prem
+        if (this.isRunMode("local")) {
```

Comment:
this is not true; when you are testing things we could imitate prod run mode on local env;

get rid of it